packaging/debian-sid/changelog

```diff
@@ -1,3 +1,125 @@
+snapd (2.62-1) unstable; urgency=medium
+
+  * New upstream release, LP: #2058277
+    - Aspects based configuration schema support (experimental)
+    - Refresh app awareness support for UI (experimental)
+    - Support for user daemons by introducing new control switches
+      --user/--system/--users for service start/stop/restart
+      (experimental)
+    - Support creating and removing recovery systems for use by factory
+      reset
+    - Enable API route for creating and removing recovery systems using
+      /v2/systems with action create and /v2/systems/{label} with action
+      remove
+    - Lift requirements for fde-setup hook for single boot install
+    - Enable single reboot gadget update for UC20+
+    - Allow core to be removed on classic systems
+    - Support for remodeling on hybrid systems
+    - Install desktop files on Ubuntu Core and update after snapd
+      upgrade
+    - Upgrade sandbox features to account for cgroup v2 device filtering
+    - Support snaps to manage their own cgroups
+    - Add support for AppArmor 4.0 unconfined profile mode
+    - Add AppArmor based read access to /etc/default/keyboard
+    - Upgrade to squashfuse 0.5.0
+    - Support useradd utility to enable removing Perl dependency for
+      UC24+
+    - Support for recovery-chooser to use console-conf snap
+    - Add support for --uid/--gid using strace-static
+    - Add support for notices (from pebble) and expose via the snapd
+      REST API endpoints /v2/notices and /v2/notice
+    - Expose AppArmor prompting experimental features supported/enabled
+      in snapd REST API endpoint /v2/system-info
+    - Add polkit authentication for snapd REST API endpoints
```

Comment:
As above, was there another PR besides #13653 which added polkit for `/v2/apps`?

Reply:
Yes, https://github.com/snapcore/snapd/pull/13367